ui/client/datasets/DataTransformation/useElwoodData.js

```diff
@@ -64,7 +65,27 @@ const useElwoodData = ({
       };
 
       if (datasetId) {
-        startElwoodJob({ requestArgs }).then((jobId) => repeatFetch(jobId));
+        startElwoodJob({ requestArgs })
```

Comment:
I'm going to pull this useElwoodData change into a new branch to start the work fixing the repeat call flow we talked about last week.

Reply:
Actually, I'm just going to bring in a skeleton version of this to avoid merge conflicts.